jsonnetfile.json

```diff
@@ -40,11 +40,11 @@
     {
       "source": {
         "git": {
-          "remote": "https://github.com/observatorium/observatorium",
+          "remote": "https://github.com/periklis/observatorium",
```

Comment:
Revert this back to upstream master when observatorium/observatorium#72 is merged.

Reply:
ready!